packages/card/src/vaadin-card.js

```diff
@@ -279,10 +280,47 @@ class Card extends ElementMixin(ThemableMixin(PolylitMixin(LitElement))) {
     `;
   }
 
+  static get properties() {
+    return {
+      /**
+       * The title of the card. When set, any custom slotted title is removed and this string-based title is used instead. If this title is used, an `aria-labelledby` attribute that points to the generated title element is set.
+       *
+       * @type {String}
```

Comment:
The `@type` in `.js` is not necessary and can be removed since we use `.d.ts` for types.

Reply:
Done.